t/cobrand/hampshire.t

```diff
@@ -0,0 +1,7 @@
+use FixMyStreet::Cobrand::Hampshire;
+use FixMyStreet::TestMech;
+
+my $hampshire = FixMyStreet::Cobrand::Hampshire->new;
+is $hampshire->base_url, 'http://www.example.org', "Hampshire returns fixmystreet base_url";
```

Comment:
Only suggestion is I would put this in an override_config section, just so you know it can't be overridden by anything. But